Python/chapter01/1.7 - Rotate Matrix/miguel_1.7_sol.py

```diff
@@ -0,0 +1,174 @@
+"""
+Python version 3.7.0
+1.7 - Rotate Matrix
+Given an image represented by an NxN matrix, where each pixel in the image is 4 bytes, write a method to rotate
+the image by 90 degrees.  Can you do this in place?
+"""
+import unittest
+from typing import List
+
+
+def rotate_matrix(matrix: List[List[int]], N: int) -> List[List[int]]:
```

Comment:
nit: you can drop the `N` parameter since it's equal to `len(matrix)`.

Reply:
Dropped